src/dmd/link.d

```diff
@@ -562,59 +562,148 @@ public int runLINK()
             argv.push("-Xlinker");
             argv.push("--gc-sections");
         }
+
+        /// Dynamic array of strings to temporarily hold accumulated,
+        /// reordered linker args
+        Strings linkerArgs;
+
+        /**
+        Checks if a C string starts with a given string
```

Comment:
Awkward. Better: `Checks if C string p starts with needle.`

Reply:
Addressed in https://github.com/dlang/dmd/pull/10341